stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/message/attachment/AttachmentUploader.kt

```diff
@@ -75,8 +75,12 @@ internal class AttachmentUploader(
             mimeType = mimeType,
             url = url,
             uploadState = Attachment.UploadState.Success,
-            type = attachmentType.toString(),
         ).apply {
+            // If attachment type was not set, set it based on this value
+            // determined by the MIME type guessed from the file's extension
+            if (type == null) {
+                type = attachmentType.toString()
+            }
```

Comment:
Previously we were overriding whatever custom `type` the user has set on the `Attachment` here